java/src/Bookings.java

```diff
@@ -2,13 +2,14 @@
 import java.util.List;
 
 public class Bookings {
-	private final List<Booking> _bookings = new ArrayList<Booking>();

```

Comment:
Don't start variable names with an underscore.

Don't need to specify the generic type again in the instantiation.